app/models/planet.py

```diff
@@ -0,0 +1,30 @@
+from app import db
+
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    description = db.Column(db.String)
+    color = db.Column(db.String)
```

Comment:
Should we be able to make planets with no name? To prevent our API from creating a planet with a `null` name, we can set that column to `nullable = False`. 

```python
name = db.Column(db.String, nullable=False)
```